roles/postgres/defaults/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 
 postgres_users: []
+postgres_backups_dir: /var/lib/postgresql-backups
```

Comment:
nit: Can you add this to `meta/main.yml` please?

Reply:
I keep forgetting about this new shiny things. Do I understand correctly that the defaults should go both here and to that file?

Reply:
So `type` and `required` are validated before role execution. That said, if you forgot to define a required variable or the type of any other is wrong, you will see a human readable error message.

`default`, on the other hand, is completely ignored at this point in time :( That why we still need `defaults/main.yml`. I want to believe it will change one day.